### PR TITLE
Warmup homework nodejs3

### PR DESCRIPTION
**Pull request by shanhaider23**

### CODE REVIEW

File: nodejs/week3/homework/warmup/index.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+app.get('/calculator/multiply',  (req, res) => {
+    const values = Object.values(req.query).flat().reduce((acc, val) => (acc) * (val))
+    res.send(` ${values}`)
+});
+app.get('/calculator/add',  (req, res) => {
+    const values = Object.values(req.query).flat().reduce((acc, val) => ((acc)*1 + (val)*1))
+    res.send(` ${values}`)
+});
+app.get('/calculator/substract',  (req, res) => {
```
Comment (cecastosic) [inline nodejs/week3/homework/warmup/index.js]:
```suggestion
app.get('/calculator/subtract',  (req, res) => {
```

**Review by cecastosic — COMMENTED** [review]
Super interesting approach 💪 
- What should happen if the query parameters are not numbers? For example /multiply?firstParam=a&secondParam=b
- What about https://github.com/HackYourFuture-CPH/node.js/blob/master/week3/homework-template/readme.md#getting-data-through-the-request-body-using-post? Have you tried to do the post requests using Postman?
- Please use Prettier to format your code https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode

File: nodejs/week3/homework/warmup/index.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    const values = Object.values(req.query).flat().reduce((acc, val) => 1*(acc - (val)))
+    res.send(` ${values}`)
+});
+app.get('/calculator/divide',  (req, res) => {
+    const values = Object.values(req.query).flat().reduce((acc, val) => 1*((acc) / (val)))
+    res.send(` ${values}`)
+});
+app.post('/calculator/add', (req, res) =>{
+    const values = Object.values(req.query).flat().reduce((acc, val) =>((acc)*1 + (val)*1))
```
Comment (cecastosic) [inline nodejs/week3/homework/warmup/index.js]:
```suggestion
    const values = Object.values(req.body).flat().reduce((acc, val) =>((acc)*1 + (val)*1))
```
Have you tested it? 🙂

Comment (shanhaider23) [reply nodejs/week3/homework/warmup/index.js]: Yes I did that I got that error (TypeError: Cannot convert undefined or null to object
    at Function.values (<anonymous>)
    at C:\Users\shanh\Desktop\hyf-homework\nodejs\week3\homework\warmup\index.js:31:25)

File: nodejs/week3/homework/warmup/index.js (hunk 1 above)
Comment (shanhaider23) [reply nodejs/week3/homework/warmup/index.js]: Thanks, for your suggestion. If query parameters are not numbers then I get NaN value. 
Yes, I used Postman for post requests.

File: nodejs/week3/homework/warmup/index.js (hunk 2 above)
Comment (cecastosic) [reply nodejs/week3/homework/warmup/index.js]: Will it work if you add app.use(express.urlencoded({extended: true}));?

Comment (shanhaider23) [reply nodejs/week3/homework/warmup/index.js]: Thanks, I implemented the change

**Pull request merged**